pkg/validator/types.go

```diff
@@ -197,3 +204,34 @@ type ResultMessage struct {
 	Type     MessageType
 	Category string
 }
+
+// ReadAuditFromFile reads the data from a past audit stored in a JSON or YAML file.
+func ReadAuditFromFile(fileName string) AuditData {
+	auditData := AuditData{}
+	oldFileBytes, err := ioutil.ReadFile(fileName)
+	if err != nil {
+		logrus.Errorf("Unable to read contents of loaded file: %v", err)
+		os.Exit(1)
+	}
+	auditData, err = ParseAudit(oldFileBytes)
+	if err != nil {
+		logrus.Errorf("Error parsing file contents into auditData: %v", err)
+		os.Exit(1)
```

Comment:
I think it would be more graceful if this ReadAuditFromFile was able to return an error that could be caught in main.go. Since this is a library file (effectively not the command runner) then it's my opinion that the main.go should be the ultimate decider of `os.Exit()` type functionality.